config/passport.js

```diff
@@ -9,36 +8,35 @@ module.exports = function(passport) {
 
   passport.deserializeUser(function(id, done) {
     User.findById(id, function(err, user) {
-        done(err, user);
+      done(err, user);
     });
   });
 
   passport.use(new GoogleStrategy({
-    clientID       : configAuth.googleAuth.clientID,
-    clientSecret   : configAuth.googleAuth.clientSecret,
-    callbackURL    : configAuth.googleAuth.callbackURL,
+    clientID     : configAuth.googleAuth.clientID,
+    clientSecret : configAuth.googleAuth.clientSecret,
+    callbackURL  : configAuth.googleAuth.callbackURL,
   },
   function(token, refreshToken, profile, done) {
     process.nextTick(function() {
       User.findOne({ 'google.id' : profile.id }, function(err, user) {
         if (err)
-            return done(err);
+          return done(err);
         if (user) {
-            return done(null, user);
+          return done(null, user);
         } else {
-          var newUser          = new User();
           newUser.google.id    = profile.id;
           newUser.google.token = token;
           newUser.google.name  = profile.displayName;
-          newUser.google.email = profile.emails[0].value;
+          newUser.google.email = profile.emails[0].value; // pull the first email
```

Comment:
This code requires a lot of knowledge of the internal structure of the `User` object - consider the addition of accessors and mutators to the `User` model to encapsulate this behavior and data.  It also appears as if you may be missing another model/class in your system: `GoogleAuthenticationInfo`.  This class should manage the `id`, `token`, `name`, etc. fields, instead of adding the knowledge of google's oauth data directly into the `User` class.